airflow/dag/serialization/serialization.py

```diff
@@ -161,7 +164,6 @@ def _serialize(cls, var):  # pylint: disable=too-many-return-statements
                 LOG.debug('Cast type %s to str in serialization.', type(var))
                 return str(var)
         except Exception:  # pylint: disable=broad-except
-            raise
```

Comment:
@ashb Removing this as if we raise excpetion it would stop the Scheduler

Reply:
Oh yes, sorry this was me testing